tt1-blocks/theme.json

```diff
@@ -175,7 +176,20 @@
 			]
 		},
 		"spacing": {
-			"customPadding": true
+			"customPadding": true,
+			"units": [
+				"px",
+				"em",
+				"rem",
+				"vh",
```

Comment:
For some reason `vh` does not show up for me:

<img width="191" alt="Screen Shot 2021-05-10 at 8 42 58 PM" src="https://user-images.githubusercontent.com/5375500/117741391-3b7b4d00-b1b7-11eb-9d0d-a99a031703d2.png">